### DISCUSSION TITLE
Qiita のトップに Qiita Blog へのリンクが欲しい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#4] [要望・アイデア・不具合]
Qiita から運営のブログを見る方法がないので、リンクをわかりやすい位置に載せてほしいです。
右上のベルからお知らせが見れますが、キャンペーンのみで Qiita の更新情報などは出てきません。

**Comment by kskwtnk** [comment]
@uhooi 
ご意見ありがとうございます！

実は……分かりづらいのですがヘッダーにあるロゴの右の「🔽」こういうアイコンを押すとドロップダウンメニューが出現し、その中の1番下にブログへのリンクが存在しています……🥲

![トップページのスクリーンショット。ロゴの右にあるディスクロージャーを展開した状態。](https://user-images.githubusercontent.com/22670303/135592837-5f4afdc6-8c9c-41dd-ab33-d8369ed74833.png)

実は現在トップページの情報をもっと分かりやすくしようというプロジェクトが走っていて、その一環としてブログへのリンクについても議論させていただきます！

**Reply by uhooi** [reply]
おわっ知りませんでした…。
ロゴの横にある▼ボタンはまったく使っていませんでした、、、

**Reply by kskwtnk** [reply]
@uhooi 

いやーこちらは、お恥ずかしい話ですが私もQiitaの開発に携わるまで全く気付いていませんでした……笑
ちょっと話逸れてしまうのですが、他のDiscussionでも話題に出していただきましたが、Qiita Blogを定期的に読んでいただけていてとても嬉しいです！

**Reply by kskwtnk** [reply]
@uhooi 

こちらのDiscussionで話していたQiita Blogへの動線ですが、次のベータ版公開にてヘッダーへ挿入されます！

以下の記事に詳細を記載していますので、よろしければ是非ご確認ください！

https://blog.qiita.com/qiita-toppage-202110-beta/
